### ISSUE TITLE
Pushgateway call registry async metrics methods

### ISSUE THREAD

**Issue by zD98** [siimon/prom-client#390]
https://github.com/siimon/prom-client/blob/25f9c4059d8a0f2d02ccfec7acfb64157c8b72bf/lib/pushgateway.js#L78

`metrics` is an async method. 

So should calling `metrics` with `async/await` ?
```javascript
const metrics = await this.registry.metrics();
req.write(metrics);
```

**Comment by zbjornson** [comment]
Yes, I think you're right. I missed two high-level functions evidently :(. PR welcome, otherwise I'll fix it in the next few days. Thanks for finding that.

**Comment by rassokhin-s** [comment]
Hey guys @zD98 @zbjornson. I have just met the same bug. Do you have any plans to merge PR and release a new version?

**Comment by neyio** [comment]
This issue still occurs at version 13.Please publish a new sub-version to resolve it, thanks.

**Comment by zbjornson** [comment]
This was fixed in #419 (thanks @adorin). Until the new version is published, you can install from github using `yarn add siimon/prom-client#83f7867` or `npm i siimon/prom-client#83f7867`.

**Comment by zbjornson** [comment]
v13.1.0 is now published with this fix!
